src/server/qgsserverinterfaceimpl.cpp

```diff
@@ -29,7 +29,7 @@ QgsServerInterfaceImpl::QgsServerInterfaceImpl( QgsCapabilitiesCache *capCache,
   mRequestHandler = nullptr;
 #ifdef HAVE_SERVER_PYTHON_PLUGINS
   mAccessControls = new QgsAccessControl();
-  mCacheManager.reset( new QgsServerCacheManager() );
+  mCacheManager = new QgsServerCacheManager();
```

Comment:
@dmarteau thanks! What was exactly the issue with the `unique_ptr` ?

Reply:
@elpaso The unique_tr require that the destructor for the template parameter to be defined. In this case there is no destructor associated to the forward declaration and result in a link error.